tests/src/Functional/SamplerFunctionalTest.php

```diff
@@ -64,18 +97,42 @@ public function testNodeDataSampling() {
     $this->createNodesOfType($nodeTypeTwo, $numberOfNodesTypeTwo, 1);
 
     $report = $this->container->get('sampler.reporter')
-      ->setAnonymize(FALSE)
+      ->anonymize(FALSE)
       ->collect()
       ->getReport();
 
     $nodeReport = $report['node'];
 
     $this->assertEquals(18, $nodeReport['base_fields']);
-    $this->assertEquals($numberOfNodesTypeOne, $nodeReport['bundles'][$nodeTypeOne]['instances']);
-    $this->assertEquals($numberOfNodesTypeTwo, $nodeReport['bundles'][$nodeTypeTwo]['instances']);
+    $this->assertEquals($numberOfNodesTypeOne, $nodeReport['bundle'][$nodeTypeOne]['instances']);
+    $this->assertEquals($numberOfNodesTypeTwo, $nodeReport['bundle'][$nodeTypeTwo]['instances']);
+
+    $this->assertEquals(2, $nodeReport['bundle'][$nodeTypeOne]['fields']);
+    $this->assertEquals(0, $nodeReport['bundle'][$nodeTypeTwo]['fields']);
+  }
+
+  /**
+   * Test sampling of node data.
+   */
+  public function testFileDataSampling() {
+    // Create test file.
+    $this->generateFile('test', 64, 10, 'text');
+    $file = File::create([
+      'uri' => 'public://test.txt',
+      'filename' => 'test.txt',
+    ]);
+    $file->setPermanent();
+    $file->save();
+
+    $report = $this->container->get('sampler.reporter')
+      ->anonymize(FALSE)
```

Comment:
Since we do not test any possibly anonymized data in this test, we could leave the anonymize on for a change.